Gruntfile.coffee

```diff
@@ -18,6 +18,31 @@ FILES =
     '!node_modules/es6-promise/dist/promise-*.min.js'
   ]
 
+INTEGRATION_SOURCES = [
+  'build/freedom/freedom-for-chrome.js'
+  'build/arraybuffers/arraybuffers.js'
+  'build/loggingprovider/loggingprovider.js'
+  'build/loggingprovider/loggingprovider.json'
+  'build/logging/logging.js'
+  'build/handler/queue.js'
+  'build/ipaddrjs/ipaddr.min.js'
+  'build/tcp/tcp.js'
+  'build/rtc-to-net/rtc-to-net.js'
+  'build/socks-common/socks-headers.js'
+  'build/socks-to-rtc/socks-to-rtc.js'
+  'build/simple-transformers/*.js'
+  'build/regex2dfa/*.js'
+  'build/utransformers/**/*.js'
+  'build/churn-pipe/churn-pipe.js'
+  'build/churn-pipe/freedom.json'
+  'build/churn/churn.js'
+  'build/webrtc/*.js'
+  # spec_base.js is listed first so that it can be used by spec.js
+  'build/integration/**/*.integration.spec_base.js'
+  'build/integration/**/*.js'
```

Comment:
Yes.  Hence the comment.  spec_base.js must be included first in main.html, and this list sets the ordering, so they must be listed separately.

Evidently the comment was not clear enough.  Improved wording suggestions welcome.